core/src/main/java/com/linecorp/armeria/common/FlagsProvider.java

```diff
@@ -989,4 +989,16 @@ default Sampler<? super RequestContext> requestContextLeakDetectionSampler() {
     default MeterRegistry meterRegistry() {
         return null;
     }
+
+    /**
+     * Returns the {@link Duration} where armeria records unlogged exceptions by default.
+     *
+     * <p>The default value of this flag is
+     * {@link DefaultFlagsProvider#defaultUnloggedExceptionReportIntervalMillis()}.</p>
+     */
+    @Nullable
+    @UnstableApi
+    default Duration defaultUnloggedExceptionReportIntervalMillis() {
```

Comment:
- `ServerBuider` uses second precision and `Flags` uses millisecond precision for the interval. Should we use seconds for consistency?
- The flag should support a JVM option via `SystemPropertyFlagsProvider`.
- We may use `long` as like other time configurations.   
```suggestion
    default long defaultUnloggedExceptionReportIntervalSeconds() {
```

Reply:
Great! I'll change it into milliseconds.

Reply:
Great. I'll modify: 
- millis -> seconds 
- support for JVM options using `SystemPropertyFlagsProvider` 
- `Duration` -> `long`

Reply:
SGTM